### PR TITLE
various coffee related objects

### PR DESCRIPTION
**Pull request by fuddl**
## hot beverage

replaces [hot beverage](https://openmoji.org/library/#search=hot%20beverage&emoji=2615). I think the current version does not convey heat and it also kind of looks like a cereal bowl.

## latte macchiato

E0C5

## roasted coffee bean

E0C6

## espresso machine

E150

## french press

E151

design based on a popular model [by bodum](https://en.wikipedia.org/wiki/Bodum)

## moka pot

E152

## milk jug

E153

## electric coffee percolator

E154

## drip coffee maker

E155

## coffee grinder

E156

based on the excellent [Commandante](https://commons.wikimedia.org/wiki/File:Commandante_Coffee_Grinder.jpg). I also considered to use a more traditional design with a barrel and drawer. what do you think?

## baristas

E185
E186
E187

These could also represent servers.

## caffiene molecule

E382

## viennese coffee house

E20C

[Viennese coffee house](https://en.wikipedia.org/wiki/Viennese_coffee_house). Usually the ground floor on mixed use building. often located at a street corner.

### CODE REVIEW

**Comment by carlinmack** [comment]
commented in the figma

**Comment by b-g** [comment]
Hi! First of all many thanks for this! Nice to see a coffee house emoji set!

But I have to say I'm a bit overwhelmed by notifications from all over the place. I really don't want to have another channel of communication for this project. In addition we should aim to stay on Github for the discussion, so that many people can cross read it as well. PLEASE!

Hence could you:
- provide an overview image
- underneath the questions
- underneath an optional public Figma link

... all here in Github ... that would extremely streamline the conversation. 

Also @dnlutz is really good with simplification. Will loop him in.

🙏

**Comment by fuddl** [comment]
>  In addition we should aim to stay on Github for the discussion, so that many people can cross read it as well. PLEASE!

agreed. I'll update this post later today

**Comment by fuddl** [comment]
@b-g updated the summary

**Comment by dnlutz** [comment]
nice! … these emoji remind me of the beginning of our project.
@johannawe is the real coffee expert ; )

**Comment by b-g** [comment]
OK. I have to say I really like them! But let's wait whether @johannawe has more feedback. Some comments from my end:

- Kaffeehaus type is really tiny. What about just using Kaffee without haus? (but not sure)
- baristas: I find the pony tail very hard to distinguish from the brown clothing
- Where is the tea :)

**Comment by fuddl** [comment]
based on @carlinmack's feedback I added more share to the coffee bean  E0C6

**Comment by dnlutz** [comment]
For the Kaffeehaus I would also just write "KAFFEE" – or even better "CAFÉ".
And honestly I don’t anybody will get the Caffeine-Molecule, so I would leave this one out.
The rest is perfect! 👏

**Comment by b-g** [comment]
And let me add ... 
- we have the 12.3 release coming up next week Friday
- please update from Master to get the new buildsystem (I changed how the openmoji.json is generated, which is relevant for everything in extra-openmoji)
- it is really a nice set! Great to have it!

**Comment by fuddl** [comment]
> For the Kaffeehaus I would also just write "KAFFEE" – or even better "CAFÉ".

_KAFFEE_ was a good fit.

> And honestly I don’t think anybody will get the Caffeine-Molecule, so I would leave this one out.
> Maybe we can transform it into a general "Molecule"-Emoji?

I like the idea that a symbol that represents molecules in general is coffeine. 

> we have the 12.3 release coming up next week Friday

I think we can manage this weekend.

File: src/extras-openmoji/food-drink/E0C5.svg [hunk 1]
```diff
@@ -0,0 +1,23 @@
+<svg id="emoji" viewBox="0 0 72 72" xmlns="http://www.w3.org/2000/svg">
```
[the remark this answers is not in the document]
Comment (fuddl) [reply src/extras-openmoji/food-drink/E0C5.svg:1]: @b-g are you sure?

1. other glasses also have no white background:
   * [milk](https://github.com/hfg-gmuend/openmoji/blob/master/src/food-drink/drink/1F95B.svg)
   * [cocktail](https://github.com/hfg-gmuend/openmoji/blob/master/src/food-drink/drink/1F378.svg)
   * [cheers](https://github.com/hfg-gmuend/openmoji/blob/master/src/food-drink/drink/1F942.svg)
   * [hourglass](https://github.com/hfg-gmuend/openmoji/blob/master/src/travel-places/time/231B.svg)
   * [drink](https://github.com/hfg-gmuend/openmoji/blob/master/src/food-drink/drink/1F379.svg)
2. at least in this case: there would be no way to distinguish between the empty part of the glass and the _filled-with-milk-foam_ part of the glass.

**Comment by fuddl** [comment]
@b-g:
> Question: Wondering why you are back to the old system of hexcodes? I thought we would go remap everything according to #203? (But happy to merge either way)

I thought there was no logical combination of unicode characters to represent either of them. Which would you suggest?

File: src/extras-openmoji/food-drink/E0C5.svg (hunk 1 above)
[the remark this answers is not in the document]
Comment (b-g) [reply src/extras-openmoji/food-drink/E0C5.svg:1]: Yes :) I would consider you examples as "bugs". The problem is that currently OpenMoji does not really work on dark backgrounds ... but it should. Removing all the "holes" might be a first step. Carlin made a nice tool to test/see all the problems:
https://hfg-gmuend.github.io/openmoji/ (try the dark toggle)

[the remark this answers is not in the document]
Comment (fuddl) [reply src/extras-openmoji/food-drink/E0C5.svg:1]: well how about a white with `.25` opacity?

**Comment by b-g** [comment]
We don’t have any opacity in OpenMoji. I am sorry! Yes ... this is all not optimal ... but we currently don’t have a better solution.

**Comment by b-g** [comment]
>  I thought there was no logical combination of unicode characters to represent either of them. Which would you suggest?

Slightly confused. Ok then the old / current way. 

Carlin, ok with this?

**Comment by carlinmack** [comment]
From the top, my thoughts are:

1. Please always align anchors with the outer strokes
image

1. Why do we have three baristas with none of them being gender neutral?

> I like the idea that a symbol that represents molecules in general is coffeine.
1. I don't really like the idea of a drug being the example of a molecule. I would prefer the much more general and 'icon'ic other symbol in the figma representing a molecule
image

>  I thought there was no logical combination of unicode characters to represent either of them. Which would you suggest? 
>Slightly confused. Ok then the old / current way.
1. yeah b-g, the ZWJ sequences only really work when there is an obvious way to represent the new emojis with a current sequence. The only one that I can think of is 🧑☕ for barista

**Comment by carlinmack** [comment]
In terms of opacity... there is actually some opacity in some like Tardis and New Moon Face, however this is only for shading (to get around the colour palette, however we should discuss this). There are no emojis that have opacity with no solid background, so E0C5 would be breaking new ground which we would need to discuss.

**Comment by fuddl** [comment]
@carlinmack what do you propose in the milk/glass scenario? should we fix this bug separately?

**Comment by carlinmack** [comment]
I'm not sure, I think either is fine, however it's most likely that this will be used on a white background and so look like it has a white background fill 🤷‍♂️

**Comment by fuddl** [comment]
sorry I tend to see only the last comment

> Why do we have three baristas with none of them being gender neutral?

I thought [that's how its done, today](https://emojipedia.org/neutral/).

**Comment by carlinmack** [comment]
which one is meant to be gender neutral though?

**Comment by fuddl** [comment]
the _person_

**Comment by carlinmack** [comment]
I think the thinner face shape you were using threw me off. I've reverted to the default face shape and it looks much more gender neutral 

image

**Comment by carlinmack** [comment]
and yes let's not include the molecule

**Comment by b-g** [comment]
@fuddl I guess on the concept all is sorted out now, or? As we are super keen to include this all in the upcoming release on Friday ... Could you aim to do the very last tiny tweaks so that we can merge it?

**Comment by fuddl** [comment]
I'll do it tonight. I promise 😁

**Comment by fuddl** [comment]
- [ ] add white fills
- [ ] remove the molecule
- [ ] re-export the person barista

**Comment by b-g** [comment]
@fuddl Great! Merged!

**Pull request merged**